tests/karate/src/test/java/be/utils/server.feature

```diff
@@ -82,6 +82,15 @@ Feature: This feature starts a server and stops it after every scenario.
     * call waitForPort
     * karate.log('Executing tests')
 
+    * def convertData =
+        """
+          function(){
+            var JsonConverter = Java.type('be.utils.JsonConverter')
+            return new JsonConverter()
+          }
+        """
+    * def converter = call convertData
```

Comment:
since all the tests are doing `convert` + `send` (with `"publish"` method) , wouldn't it make sense to define an API here that allows you to just `publish(message, id)` ?